presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/MetadataHandle.java

```diff
@@ -60,7 +62,10 @@ public void addTableColumnMetadata(QualifiedObjectName tableName, Future<TableCo
 
     public Optional<ViewDefinition> getViewDefinition(QualifiedObjectName viewName)
     {
-        checkState(viewDefinitions.containsKey(viewName), "View " + viewName + " not found, the available view names are " + viewDefinitions.keySet());
+        if (!viewDefinitions.containsKey(viewName)) {
+            throw new PrestoException(VIEW_NOT_FOUND, "View " + viewName + " not found, the available view names are " + viewDefinitions.keySet());
```

Comment:
Let's follow the same pattern we use other places for the exception message as below:

`throw new PrestoException(VIEW_NOT_FOUND, "View %s not found, the available view names are: %s", viewName, viewDefinitions.keySet());`